mesonbuild/programs.py

```diff
@@ -266,7 +268,7 @@ def _search_windows_special_cases(self, name: str, command: str) -> T.List[T.Opt
             commands = self._shebang_to_cmd(command)
             if commands:
                 return commands
-            return [None]
```

Comment:
You may find yourself biting off more than you want to chew with this one, unfortunately